From: g0t mi1k <have.you.g0tmi1k@gmail.com>
Date: Fri, 5 Jul 2024 10:25:22 +0100
Subject: Fix arm64 support

Forwarded: https://github.com/jaygreig86/dmitry/pull/15
---
 src/includes/dmitry.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/includes/dmitry.h b/src/includes/dmitry.h
index 83ee5f8..738cec4 100644
--- a/src/includes/dmitry.h
+++ b/src/includes/dmitry.h
@@ -1,4 +1,4 @@
-char optchar;
+int optchar;
 int options[10];
 int ctr;
 extern void print_line();
