Description: Avoid conflicting prototypes for file_open().
Author: Petter Reinholdtsen
Bug-Debian: https://bugs.debian.org/746769
Forwarded: https://github.com/jaygreig86/dmitry/pull/17
Last-Update: 2024-07-12
---
--- dmitry-1.3a.orig/src/includes/iwhois.h
+++ dmitry-1.3a/src/includes/iwhois.h
@@ -29,7 +29,7 @@ int ip_format_buff();
 int linetodo;   /* This is used to symbolise to ip_format_buff() that there is a line still to format */
 char query[26];
 extern char outputfile[64];
-extern void file_open();
+extern int file_open();
 extern void file_close();
 
 char *ip_whois_hosts[] = {
--- dmitry-1.3a.orig/src/includes/mailsearch.h
+++ dmitry-1.3a/src/includes/mailsearch.h
@@ -8,7 +8,7 @@ extern void sendData();
 extern int get_host();
 extern void tcp_sockdcon();
 extern char outputfile[64];
-extern void file_open();
+extern int file_open();
 extern void file_close();
 extern void print_line();
 extern int tcp_sock;
--- dmitry-1.3a.orig/src/includes/nwhois.h
+++ dmitry-1.3a/src/includes/nwhois.h
@@ -15,7 +15,7 @@ char td[10];
 char query[128];
 int linetodo;   /* This is used to symbolise to nic_format_buff() that there is a line still to format */
 extern char outputfile[64];
-extern void file_open();
+extern int file_open();
 extern void file_close();
 
 
--- dmitry-1.3a.orig/src/includes/subsearch.h
+++ dmitry-1.3a/src/includes/subsearch.h
@@ -9,7 +9,7 @@ extern void sendData();
 extern int get_host();
 extern void tcp_sockdcon();
 extern char outputfile[64];
-extern void file_open();
+extern int file_open();
 extern void file_close();
 extern void print_line();
 extern int tcp_sock;
